src/main/webapp/WEB-INF/views/cars/drivers/add.jsp

```diff
@@ -0,0 +1,14 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Add driver to car</title>
+</head>
+<body>
+<h1>Add driver to car</h1>
+<form method="post" action="${pageContext.request.contextPath}/cars/drivers/add">
+  Car id: <input type="text" name="car_id"><br>
+  Driver id: <input type="text" name="driver_id"><br>

```

Comment:
<img width="978" alt="image" src="https://user-images.githubusercontent.com/58032907/203354715-6376f7c0-4eb2-4a8e-8fed-1ee64bfa030f.png">
type shouldn't be `text`